dias/utilities.py

```diff
@@ -0,0 +1,57 @@
+import re
+from pathlib import Path
+import subprocess
+
+
+SIZE_POWERS = ["B", "K", "M", "G", "T", "P"]
```

Comment:
What stands behind these letters?

Reply:
Prefixes of B - 1024**0, KiB - 1024**1, MiB - 1024**2, GiB - 1024**3, etc.
Used for extracting file sizes (in bytes) from file listings, I think.